docs/topics/feed-exports.rst

```diff
@@ -268,6 +268,83 @@ in multiple files, with the specified maximum item count per file. That way, as
 soon as a file reaches the maximum item count, that file is delivered to the
 feed URI, allowing item delivery to start way before the end of the crawl.
 
+.. _post-processing:
+
+Post-Processing
+===============
+
+.. versionadded:: VERSION
+
+Scrapy provides an option to activate plugins to post-process items before
+they are exported to feed storages. In addition to using builtin plugins, users
+can create their own plugins. 
+
+These plugins can be activated through ``postprocessing`` :ref:`option of a feed <feed-options>`.
+Parameters to plugins can be passed through the feed settings where the key will be the parameter name
+and the value will be the parameter value. These parameters will be made accessible to the plugin
+through the ``feed_options`` parameter which will be passed to their constructor method.
+
+As the data is passed from one plugin to another, users must ensure the order of plugins do not
+cause a plugin to breakdown because of some unprocessable data.
+
+Each plugin is a Python class that must implement the following methods:
```

Comment:
```suggestion
Each plugin is a class that must implement the following methods:
```

I’m not saying that no one will ever try to define a plugin in a different language. I’m saying that, when they realize why their plugin is not working, they will possibly feel ashamed and they will definitely not request that we explicitly say Python here to clarify :slightly_smiling_face: